libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/RoutesRenderedCallback.kt

```diff
@@ -0,0 +1,82 @@
+package com.mapbox.navigation.ui.maps.route.line.api
+
+import com.mapbox.maps.MapboxMap
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.base.route.NavigationRoute
+
+/**
+ Callback to be notified of rendered routes.
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+interface RoutesRenderedCallback {
+
+    /**
+     * Map instance where the routes are expected to be rendered.
+     */
+    val map: MapboxMap
```

Comment:
Maybe it should be a parameter rather than a part of the callback?

Reply:
It can, I just thought that it always comes with a callback, so why not store it there? I don't have a strong opinion here.
@LukasPaczos any thoughts?

Reply:
There's a small chance of a memory leak if someone holds onto the callback object beyond the lifecycle of the map. It's an edge-case but it's still a risk we introduce with this type of an API. If we don't have a strong opinion, then passing a map as a `MapboxRouteLineView` parameter might be better because then we can make `RoutesRenderedCallback` a SAM interface.

Reply:
Addressed.